StartCgnActivation/handler.ts

```diff
@@ -88,20 +89,24 @@ const mapOrchestratorStatus = (
  * @param fiscalCode: the citizen's fiscalCode
  */
 const getCgnExpirationDataTask = (
-  fiscalCode: FiscalCode
+  fiscalCode: FiscalCode,
+  cgnBetaTestUpperBoundAge: NonNegativeInteger | undefined
 ): TaskEither<
   IResponseErrorInternal | IResponseErrorForbiddenNotAuthorized,
   Date
 > =>
-  checkCgnRequirements(fiscalCode).foldTaskEither<
+  checkCgnRequirements(fiscalCode, cgnBetaTestUpperBoundAge).foldTaskEither<
```

Comment:
Can we use `chain` instead of `foldTaskEither`?
```typescript
checkCgnRequirements(fiscalCode, cgnBetaTestUpperBoundAge)
    .mapLeft<IResponseErrorInternal | IResponseErrorForbiddenNotAuthorized>(_ =>
      ResponseErrorInternal("Cannot perform CGN Eligibility Check")
    )
    .chain(
      fromPredicate(
        isElegible => isElegible === true,
        () => ResponseErrorForbiddenNotAuthorized
      )
    )
    .chain(_ =>
      extractCgnExpirationDate(
        fiscalCode,
        cgnBetaTestUpperBoundAge
      ).mapLeft(() =>
        ResponseErrorInternal("Cannot perform CGN Eligibility Check")
      )
    );
```